### PR TITLE
Migrate to using components from the component library and minor UI fixes

### PR DESCRIPTION
**Pull request by robiot**

### CODE REVIEW

**Comment by robiot** [comment]
image
image
image
image
image
image
image
image
image

File: apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx [hunk 1]
```diff
@@ -171,46 +171,40 @@ export const CreateKeyModal: FC<{ onClose: () => void }> = ({ onClose }) => {
...
                     <div>
-                        <span className="block text-base font-medium text-neutral-900 dark:text-neutral-300 mb-2">
-                            Key Permissions
-                        </span>
+                        <div className={titleClass}>Key Permissions</div>
                         <div className="grid items-center gap-2 text-neutral-500">
                             <Checkbox
                                 id="Full Access"
-                                label="Full Access"
+                                // {...register('Full Access')}
```
Comment (lucemans) [inline apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx:194]: ?

File: apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx [hunk 2]
```diff
@@ -171,46 +171,40 @@ export const CreateKeyModal: FC<{ onClose: () => void }> = ({ onClose }) => {
...
                                 <Checkbox
                                     id={perm}
                                     key={perm}
-                                    label={perm}
                                     register={register}
-                                    disabled={watch('Full Access')}
-                                />
+                                    isDisabled={watch('Full Access')}
+                                >
+                                    {perm}
```
Comment (lucemans) [inline apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx:206]: perm?

File: apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx [hunk 3]
```diff
@@ -221,35 +215,32 @@ export const CreateKeyModal: FC<{ onClose: () => void }> = ({ onClose }) => {
...
                     <div>
-                        <span className="block text-base font-medium text-neutral-900 dark:text-neutral-300 mb-2">
-                            Settings
-                        </span>
+                        <div className={titleClass}>Settings</div>
                         <div className="block w-full">
                             <Checkbox
                                 id="permanent"
-                                label="Permanent Key"
+                                // label=""
```
Comment (lucemans) [inline apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx:222]: ?

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/CVS": true,
+    "**/.DS_Store": true,
+    "**/Thumbs.db": true,
     "node_modules": true,
     "packages/groot/tmp": true,
     "pnpm-lock.yaml": true,
-    "**/node_modules": true,
+    "**/node_modules": false,
```
Comment (Antony1060) [inline .vscode/settings.json:12]: Erm, any reason for this?

File: apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx [hunk 5]
```diff
@@ -262,18 +253,17 @@ export const CreateKeyModal: FC<{ onClose: () => void }> = ({ onClose }) => {
...
-                            isValid && !noPerms
-                                ? isSubmitting
-                                    ? 'Pending...'
-                                    : 'Create 🔑'
-                                : 'Select Permissions'
-                        }
-                    />
+                        isDisabled={isSubmitting || !isValid || noPerms}
+                        loading={isSubmitting}
+                        // variant="primary"
```
Comment (lucemans) [inline apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx:258]: ?

**Review by Antony1060 — APPROVED** [review]
👀

File: packages/components/src/components/Common/TextField.tsx [hunk 6]
```diff
@@ -1,17 +1,22 @@
...
 export interface TextFieldProperties
-    extends Omit<AriaTextFieldProps, 'onChange'>,
-        FocusableOptions {
+    extends AriaTextFieldProps,
+        FocusableOptions,
+        BaseRegisterComponent {
     className?: string;
 
-    onChange?: ChangeEventHandler<HTMLInputElement>;
+    // onChange?: ChangeEventHandler<HTMLInputElement>;
```
Comment (lucemans) [inline packages/components/src/components/Common/TextField.tsx:19]: ?

File: packages/components/src/components/Common/TextField.tsx [hunk 7]
```diff
@@ -47,7 +52,7 @@ const TextField: FC<TextFieldProperties> = (properties) => {
             <input
                 {...inputProps}
                 // onChange={properties.onChange}
```
Comment (lucemans) [inline packages/components/src/components/Common/TextField.tsx:54]: ?

**Review by lucemans — CHANGES_REQUESTED** [review]
Please bring Container back rofl. And remove the comments

File: .vscode/settings.json (hunk 4 above)
Comment (lucemans) [reply .vscode/settings.json:12]: Please change back to true

Comment (robiot) [reply .vscode/settings.json:12]: Sure thing

File: apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx (hunk 2 above)
Comment (robiot) [reply apps/app/src/components/CreateKeyModal/CreateKeyModal.tsx]: Supposedly permissions. Just moved it from the label parameter to children.

**Review by svemat01 — APPROVED** [review]
LGTM

**Review by lucemans — APPROVED** [review]
LGTM

**Pull request merged**